ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-next-periodics.yaml

```diff
@@ -0,0 +1,149 @@
+periodics:
```

Comment:
Introducing a `release-next-periodics` should be fine, right?

Reply:
Yes, I think it is fine, but the `release-next-periodics` is same with the release-next job like https://github.com/openshift-knative/serving/pull/123 ?